src/ui/components/results/directanswercomponent.js

```diff
@@ -93,19 +93,55 @@ export default class DirectAnswerComponent extends Component {
 
     // For WCAG compliance, the feedback should be a submittable form
     DOM.on(this._formEl, 'submit', (e) => {
-      let formEl = e.target;
-      let checkedValue = DOM.query(formEl, 'input:checked').value === 'true';
+      const formEl = e.target;
+      const checkedValue = DOM.query(formEl, 'input:checked').value === 'true';
 
       this.reportQuality(checkedValue);
       this.updateState({
-        'feedbackSubmitted': true
+        feedbackSubmitted: true
       });
     });
 
     // Is this actually necessary? I guess it's only necessary if the
     // submit button is hidden.
     DOM.on(this._thumbsUpSelector, 'click', () => { DOM.trigger(this._formEl, 'submit'); });
     DOM.on(this._thumbsDownSelector, 'click', () => { DOM.trigger(this._formEl, 'submit'); });
+
+    const rtfElement = DOM.query(this._container, '.js-yxt-rtfValue');
+    rtfElement && this._handleRtfClickAnalytics(rtfElement);
```

Comment:
Could we move `DOM.on(rtfElement, 'click', eventHandler);` up here, and change the eventHandler method in _handleRtfClickAnalytics to just be the main method? 
Right now _handleRtfClickAnalytics isn't really an event handler it's an event registerer. We could also use the arrow function to get rid of const self = this like
`rtfElement && DOM.on(rtfElement, 'click', e => this._handleRtfClickAnalytics(e, rtfElement);`

*same for the click analytics in cardcomponent.js